send_comments.py

```diff
@@ -0,0 +1,82 @@
+import requests
+import os
```

Comment:
High severity: Calling response.json() without guarding can raise ValueError if the response body is not JSON. Replace with safe parsing and proper error handling (score: 8/10).
```suggestion
try:
    content = response.json()
except ValueError:
    content = response.text

if not response.ok:
    # Log status and handle retry or raise exception
    logging.error(f"Request failed with status {response.status_code}: {content}")
    raise Exception("Request failed")
```